test/ui_test.exs

```diff
@@ -0,0 +1,74 @@
+defmodule UITest do
+  use ExUnit.Case
+
+  import ExUnit.CaptureIO
+
+  test "introduces game" do
+    assert capture_io(fn ->
+      UI.introduce_ttt
+    end) == "THIS IS *** TIC-TAC-TOE ***\n"
+  end
+
+  test "asks who starts" do
+    assert capture_io([input: "x\n"], fn ->
+      UI.prompt_for_starter
+    end) == "Who starts? [enter X or O]\n"
+  end
+
+  test "registers and formats starter mark chosen by user" do
+    assert capture_io([input: "x\n"], fn ->
+      assert UI.prompt_for_starter == "X"
+    end)
+  end
+
+  test "prints board" do
+    assert capture_io(fn ->
+      board = %Board{size: 3, cells: Board.create_cells(3)}
```

Comment:
👋 How come we have to use `3` twice? Is the `:size` somehow connected to the _created cells_? 
I feel like this would be a valid place to create function that gave you a fresh, empty board of a given size 🛩